.gitignore

```diff
@@ -1,14 +1,5 @@
-node_modules
-yarn-error.log
-build
-dist
-junit.xml
 .DS_Store
-docs
-
-# Yarn
-.yarn/*
```

Comment:
huh, how did I end up with a `.yarn/cache` locally? now when I rm it and re-run yarn, it doesn't recreate it...

Reply:
you're using yarn 2 somehow